src/services/sync/strategies/WebUntis/WebUntisBaseSyncer.js

```diff
@@ -0,0 +1,781 @@
+const { BadRequest } = require('@feathersjs/errors');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['BadRequest' is assigned a value but never used.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=4061937)